rke/k8s_rke_system_images.go

```diff
@@ -5344,6 +5344,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled out of band post v2.6.0
```

Comment:
Those will be in 2.6.0, not post, right?

Reply:
Right, I need to update those. It was initially OOB, but that changed today.